src/services/client-data-mapping-service.js

```diff
@@ -1,13 +1,14 @@
 import uniqueId from '../utils/unique-id.js';
 import cloneDeep from '../utils/clone-deep.js';
+import UserStore from '../stores/user-store.js';
 import privateData from '../domain/private-data.js';
-import UserService from '../services/user-service.js';
+import { extractUserIdsFromDocsOrRevisions } from '../domain/data-extractors.js';
```

Comment:
Extracted this out of `user-service` into a simple domain helper, and with this:
- the `client-data-mapping-service` is decoupled from the `user-service`
- the `export-service` is decoupled from the `user-service`

Reply:
The remaining service to service dependencies now are:

<img width="323" alt="Screen Shot 2022-02-23 at 11 35 36" src="https://user-images.githubusercontent.com/8189923/155302911-3b0ac0ee-6c0f-47b3-ac52-051e688e04b5.png">

Reply:
Nice, this is a good pattern for sharing code without getting into interdependencies. :heart: